### PR TITLE
[LAB-18] Added course by owner id endpoint

### LINKED ISSUE #18: Get Courses by owner id

### PR DESCRIPTION
**Pull request by Marko-Shevchuk**
closes #18

### CODE REVIEW

File: src/main/java/com/pnu/lab/control/labcontrol/api/CourseController.java [hunk 1]
```diff
@@ -47,6 +48,11 @@ public List<CourseMember> getCourseMembers(@PathVariable String id) {
         return courseService.getCourseMembers(id);
     }
 
+    @GetMapping("/listByOwner")
```
Comment (ArsenOOO7) [inline src/main/java/com/pnu/lab/control/labcontrol/api/CourseController.java:51]: list/byOwner

File: src/main/java/com/pnu/lab/control/labcontrol/service/CourseService.java [hunk 2]
```diff
@@ -29,6 +29,10 @@ public List<CoursePreviewDto> getUserCourseList(UserCourseListRequest request) {
         return repository.getUserCourseList(request);
     }
 
+    public List<CoursePreviewDto> getOwnerCourseList(String ownerId) {
+        return repository.getOwnerCourseList(ownerId);
```
Comment (ArsenOOO7) [inline src/main/java/com/pnu/lab/control/labcontrol/service/CourseService.java:33]: same

File: src/main/java/com/pnu/lab/control/labcontrol/api/CourseController.java (hunk 1 above)
Comment (ArsenOOO7) [reply src/main/java/com/pnu/lab/control/labcontrol/api/CourseController.java:51]: and the method naming

**Review by ArsenOOO7 — APPROVED** [review]

**Review by JJerome-NM — APPROVED** [review]
I trust you

**Pull request merged**